src/contract.rs

```diff
@@ -1,35 +1,51 @@
+#[cfg(not(feature = "library"))]
+use cosmwasm_std::entry_point;
 use cosmwasm_std::{
-    coins, to_binary, BankMsg, Binary, CosmosMsg, Deps, DepsMut, Env, HandleResponse, InitResponse,
-    MessageInfo, Order, StdResult, Storage,
+    coins, to_binary, BankMsg, Binary, Deps, DepsMut, Env, MessageInfo, Order, Response, StdResult,
+    Storage, SubMsg,
 };
 use drand_verify::{derive_randomness, g1_from_variable, verify};
 
-use crate::errors::{HandleError, QueryError};
+use crate::errors::ContractError;
 use crate::msg::{
-    BountiesResponse, Bounty, GetResponse, HandleMsg, InitMsg, LatestResponse, QueryMsg,
+    BountiesResponse, Bounty, ConfigResponse, ExecuteMsg, GetResponse, InstantiateMsg,
+    LatestResponse, QueryMsg,
 };
 use crate::state::{
     beacons_storage, beacons_storage_read, bounties_storage, bounties_storage_read, config,
     config_read, Config,
 };
 
-pub fn init(deps: DepsMut, _env: Env, _info: MessageInfo, msg: InitMsg) -> StdResult<InitResponse> {
+use cw2::set_contract_version;
+
+const CONTRACT_NAME: &str = "crates.io:rand";
+const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
+
+#[cfg_attr(not(feature = "library"), entry_point)]
```

Comment:
This contract does not intend to be a library, so it can just be

```suggestion
#[entry_point]
```

Reply:
Fixed